gateway/src/main/java/com/hurence/webapiservice/historian/impl/SolrHistorianServiceImpl.java

```diff
@@ -213,6 +217,57 @@ private SolrQuery buildTimeSeriesChunkQuery(JsonObject params) {
         query.setRows(params.getInteger(MAX_TOTAL_CHUNKS_TO_RETRIEVE_REQUEST_FIELD, 50000));
         return query;
     }
+    private SolrQuery buildAnnotationQuery(JsonObject params) {
+        StringBuilder queryBuilder = new StringBuilder();
+        Long from = params.getLong(FROM_REQUEST_FIELD, null);
```

Comment:
Specifying null as default value is useless as this is the same than using params.getLong(FROM_REQUEST_FIELD)